src/components/storage/storageSrv.js

```diff
@@ -233,9 +233,12 @@
 
                 return this.get(path).then(function (pathValue) {
                     self.adapter.onEvent('value', pathValue.$$path, function (serverValue) {
+                        if (typeof serverValue !== 'object'){
+                            $log.error('getAndBindToServer Fn support only object value');
```

Comment:
I would have change it to "getAndBindToServer fn supports only binding to an object (and not other value)"